tchannel-core/src/main/java/com/uber/tchannel/api/SubChannel.java

```diff
@@ -226,13 +226,13 @@ public boolean sendOutRequest(OutRequest outRequest) {
         return sendRequest(request, host, port);
     }
 
-    public ListenableFuture<RawResponse> send(
+    public TFuture<RawResponse> send(
```

Comment:
TFuture is inherited from ListenableFuture. So, it is not breaking change. However, new implementations should use TFuture instead of ListenableFuture.